### PR TITLE
Update geoserver

### PR DESCRIPTION
**Pull request by julian1**

### CODE REVIEW

**Comment by julian1** [comment]
hopefully the last one, and we can actually do some proper testing!

File: emii/pom.xml [hunk 1]
```diff
@@ -90,8 +100,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
                 <configuration>
-                    <source>1.5</source>
-                    <target>1.5</target>
+                     <source>1.5</source>
```
Comment (danfruehauf) [inline emii/pom.xml]: What happened here?

File: emii/pom.xml [hunk 2]
```diff
@@ -7,56 +7,66 @@
...
         <artifactId>geoserver</artifactId>
         <version>1.0.0</version>
-        <relativePath>../../pom.xml</relativePath>
     </parent>
 
     <dependencies>
+
+        <dependency>
+            <groupId>org.geoserver.web</groupId>
+            <artifactId>gs-web-app</artifactId>
```
Comment (danfruehauf) [inline emii/pom.xml]: So why is this still here if we're taking geoserver as submodule?

Comment (julian1) [inline emii/pom.xml]: It's either for the overlay manager which is still used, or for the custom xslt code.

File: emii/pom.xml (hunk 1 above)
Comment (julian1) [inline emii/pom.xml]: Maybe Craig's Eclipse?

File: emii/pom.xml (hunk 2 above)
Comment (danfruehauf) [inline emii/pom.xml]: Overlay manager?

**Comment by danfruehauf** [comment]
I'm merging this, but I really want to be able to get back to source `gs-web-app` without the need of buliding geoserver. I hope we agree on that.

File: emii/pom.xml (hunk 2 above)
Comment (julian1) [inline emii/pom.xml]: yes.

Comment (julian1) [inline emii/pom.xml]: We should get rid of it. But I don't want to play with it more right now.

Comment (danfruehauf) [inline emii/pom.xml]: You assume I know what's the overlay manager. I don't.

What is it?

Comment (julian1) [inline emii/pom.xml]: It munges the jars, to insert our custom class files and context for the csv with metadata header.

**Pull request merged**